### DISCUSSION TITLE
Why does this repository use the "Discussions" feature?

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#35] [General]
Maybe it's just me, but I've never seen the "Discussions" feature enabled in any other repository except this one.

Just out of curiosity, why don't we use "Issues" instead since every other repository seems to use it over the "Discussions" feature?

**Comment by mayrinck** [comment]
Because issues usually are more directed to code problems/bugs/requests, and this repo has no code at all, so it was a choice that made sense at the time...

**Reply by JuliaAgostino** [reply]
> Because issues usually are more directed to code problems/bugs/requests, and this repo has no code at all, so it was a choice that made sense at the time...

Fair point, but wouldn't it make sense as we're technically maintaining the applications in their packaged state and having to constantly update them whenever a new version comes out?

Of course, this is your repository. Totally up to you... 🙂
